Sources/AWSSigner/signer.swift

```diff
@@ -120,7 +120,12 @@ public struct AWSSigner {
         var date : String { return String(datetime.prefix(8))}
 
         init(url: URL, method: HTTPMethod = .GET, headers: HTTPHeaders = HTTPHeaders(), body: BodyData? = nil, bodyHash: String? = nil, date: String, signer: AWSSigner) {
-            self.url = url
+            if url.absoluteString.hasSuffix("amazonaws.com") {
+                //URL has to have trailing slash
```

Comment:
This makes the assumption the URL ends in `amazonaws.com`. Could we not use `url.path == ""`

Reply:
🤦‍♂️Yeah that makes more sense. Thanks.